app/models/github/issue_label.rb

```diff
@@ -10,20 +10,48 @@ def self.for_type(type, val)
     return unless TYPES.include?(type)
     return unless TYPES[type].include?(val)
 
-    "#{type_prefix(type)}#{val}"
+    "x:#{type}/#{val}"
   end
 
-  def of_type?(type)
-    name.start_with?(Github::IssueLabel.type_prefix(type))
+  def of_type?(type_to_check)
```

Comment:
I think there's two methods getting pulled together here:
- `of_type?`
- `valid_for_type?`

The second can always call the first, but I think seperating them will make this more readable.